### DISCUSSION TITLE
trying but i do not understand

### DISCUSSION THREAD

**Discussion by ldijkman** [mkslanc/ace-linters#82] [Q&A]
treid your example
copied the example from the page
but i think it did not work 

https://plnkr.co/plunk/jdFyYMSzz00TAXYP

think your code beautifier is working nice what i have seen in your demo
on
https://mkslanc.github.io/ace-playground/#hello-world
CTRL+SHIFT+b works better as ace beatifier

would like to  use it on 
https://ldijkman.github.io/Ace_Seventh_Heaven/Hell.html

**Comment by mkslanc** [comment]
Hi, @ldijkman ! you just need to add 
```html
<script src="https://www.unpkg.com/ace-builds@latest/esm-resolver.js"></script>
```
to include all modes/themes, etc.
or 
```html
<script src="https://www.unpkg.com/ace-builds@latest/src-noconflict/mode-html.js"></script>
```
for html (for example)

and use mode name in ace editor initialization by adding option  - `mode: "ace/mode/html"`
